SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -48,6 +48,9 @@ bool USpatialNetDriver::InitBase(bool bInitAsClient, FNetworkNotify* InNotify, c
 	bConnectAsClient = bInitAsClient;
 	bAuthoritativeDestruction = true;
 
+	TypebindingManager = NewObject<USpatialTypebindingManager>();
```

Comment:
Is it ok not to check if it's `nullptr`?

Reply:
This is how it used to be, should always be nullptr as this is now back in InitBase()